src/IIIFPresentation/API/KnownCollections.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace API;
+
+public static class KnownCollections

```

Comment:
I think this might be (correctly) duplicating functionality of the `Test.Helpers.Helpers.RootCollection` class - worth removing that class now this has been added?